Week 2/Homework2.html

```diff
@@ -0,0 +1,128 @@
+<html>
+	<head>
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+	</head>
+	<body>
+		<br>
+		Book: <input type="text" id="book"/> <br>
+		Author: <input type="text" id="bookauthor"/> <br>
+		Price: <input type="text" id="bookprice"/> <br>
+		<button onclick="submitBook();">Submit Book</button> <br>
+		<br>
+		Library <input type="text" id="library"/> <br>
+		Place <input type="text" id="libraryplace"/> <br>
+		Books <input type="text" id="librarybooks"/> <br>
+		<button onclick="submitLibrary();">Submit Library</button>
+		<br>
+		<button onclick="output();">UpdateList</button>
+		<br>
+		<span id="libraryoutput">LIBRARY <br></span>
+		<br>
+		<span id="bookoutput">BOOK<br><br></span>
+		
+	</body>
+	
+	<script src="https://cloudboost.io/js-sdk/cloudboost.js"></script>
+	<script>
+	CB.CloudApp.init('njoumzczehoq', 'b7cc60b7-a1ad-4a0e-8d1c-aceab6dcb4f4');
+	let thelibrary = new CB.CloudObject("Library");
```

Comment:
You shouldn't declare global variables and then use it inside the class. This is actually what will happen...

```js
const theLibrary = new CB.CloudObject("Library");
const lib = new Library("Lib 1", "Manhattan");
const lib2 = new Library("Lib 2", "Queens");
```

Both constructors will actually modify the CloudBoost object, however each instance of a class should contain it's own cloudboost object.

```js
class Library {
  constructor(name, place) {
    this.cloudObject = new CloudObject("Library");
  }
}
```